packages/react/src/modal/ModalDialog.tsx

```diff
@@ -58,7 +58,11 @@ export const ModalDialog = ({
 					margin: '0 auto',
 					minHeight: '100vh',
 					animation: `${animateSlideInUp} ${tokens.transition.duration}ms ${tokens.transition.timingFunction}`,
+					'@supports (min-height: 100dvh)': {
+						minHeight: '100dvh',
+					},
 					[tokens.mediaQuery.min.sm]: {
+						borderRadius: tokens.borderRadius,
```

Comment:
I feel like this should be at `md` breakpoint (768px) since a common use-case is going to be wide tables that encourage users to rotate their phones to landscape mode. What do you think?

Reply:
I think that can wait for a separate discussion and PR. The designers may have mandated this. I just fixed what was there